lusid-jam/refreshing_token.py

```diff
@@ -0,0 +1,31 @@
+from collections import UserString
+from datetime import datetime, timedelta, time
+import os
+
+class RefreshingToken(UserString):
+
+    def __init__(self, access_token=os.path.join(os.getenv("FBN_ACCESS_TOKEN_FILE"))):
```

Comment:
Is os.path.join the best thing to call here? We will just be providing the absolute path to the secrets file so I'm not sure if join adds any value as we are not joining it with anything?

Reply:
Agreed. The os.path.join method was a typo. I have removed that.